src/pages/Dashboard/Automations/AddAutomationCard.vue

```diff
@@ -29,6 +33,10 @@ export default {
         openDuration: { name: 'openDuration', complete: false },
         selectDoThis: { name: 'selectDoThis', complete: false }
       },
+      hookTypes: [
+        { type: 'flow', label: 'flow run', permission: 'feature:hooks' },
```

Comment:
`feature:hooks` dropped in https://github.com/PrefectHQ/cloud/pull/3302

Reply:
Ah yes, this has been updated already in #751 👍 , maybe i should close this PR in favor of that one

Reply:
Mm yeah that looks good on that PR 🤷